src/rendering/vcFenceRenderer.cpp

```diff
@@ -60,44 +60,60 @@ static vcTexture *gSolidTexture = nullptr;
 static vcTexture *gDiagonalTexture = nullptr;
 
 udResult vcFenceRenderer_CreateSegmentVertexData(vcFenceRenderer *pFenceRenderer, vcFenceSegment *pSegment);
+udResult vcFenceRenderer_Init();
+udResult vcFenceRenderer_Destroy();
 
-void vcFenceRenderer_Init()
+udResult vcFenceRenderer_Init()
 {
+  udResult result;
   gRefCount++;
-  if (gRefCount == 1)
-  {
-    vcTexture_CreateFromFilename(&gArrowTexture, "asset://assets/textures/fenceArrow.png", nullptr, nullptr, vcTFM_Linear, true);
-    vcTexture_CreateFromFilename(&gGlowTexture, "asset://assets/textures/fenceGlow.png", nullptr, nullptr, vcTFM_Linear, true);
-    vcTexture_CreateFromFilename(&gDiagonalTexture, "asset://assets/textures/fenceDiagonal.png", nullptr, nullptr, vcTFM_Linear, true);
+  static const uint32_t whitePixel[] = { 0xffffffff };
 
-    static const uint32_t solidPixels[] = { 0xffffffff };
-    vcTexture_Create(&gSolidTexture, 1, 1, solidPixels);
-  }
+  UD_ERROR_IF(gRefCount != 1, udR_Success);
+
+  UD_ERROR_IF(!vcTexture_CreateFromFilename(&gArrowTexture, "asset://assets/textures/fenceArrow.png", nullptr, nullptr, vcTFM_Linear, true), udR_InternalError);
+  UD_ERROR_IF(!vcTexture_CreateFromFilename(&gGlowTexture, "asset://assets/textures/fenceGlow.png", nullptr, nullptr, vcTFM_Linear, true), udR_InternalError);
+  UD_ERROR_IF(!vcTexture_CreateFromFilename(&gDiagonalTexture, "asset://assets/textures/fenceDiagonal.png", nullptr, nullptr, vcTFM_Linear, true), udR_InternalError);
+
+  UD_ERROR_CHECK(vcTexture_Create(&gSolidTexture, 1, 1, whitePixel));
+
+  result = udR_Success;
+epilogue:
+
```

Comment:
You've got the spacing around the wrong way here. Usually we have the empty line _before_ the epilogue.

Reply:
Arg! Fixing.